vhost_user_block/src/lib.rs

```diff
@@ -270,8 +270,8 @@ impl VhostUserBackend for VhostUserBlkBackend {
         debug!("event received: {:?}", device_event);
 
         match device_event {
-            q if device_event < self.config.num_queues => {
-                let mut vring = vrings[q as usize].write().unwrap();
+            _q if device_event < self.config.num_queues => {
```

Comment:
Why this change?

Reply:
> Please split this commit into smaller changes:
> 
> * Introduce infrastructure into vhost-user-backend to allow the backends to control their worker thread
> * Arc
> * Port -net to use it
> 
> With the exception of if you change function names I don't think there should be changes to -block and -fs.

The three items above are the current existing commits, you mean to split them into smaller ones? While there are new methods added in VhostUserBackend to acquire backends' information to support multiple threads, since -block and -fs have different behavior on that, changes are required to provide concrete implementation respectively. Let me know if you still think the changes is not needed.